src/lib.rs

```diff
@@ -8,9 +8,24 @@ pub use node::{chain_spec, Builder as NodeBuilder, Info as FarmerInfo, Mode as N
 pub use parse_ss58::Ss58ParsingError;
 
 use derive_more::{Deref, DerefMut};
+use serde::{Deserialize, Serialize};
 use subspace_core_primitives::PUBLIC_KEY_LENGTH;
 
-#[derive(Debug, Default, Copy, Clone, PartialEq, Eq, Ord, PartialOrd, Hash, Deref, DerefMut)]
+#[derive(
+    Debug,
+    Default,
+    Copy,
+    Clone,
+    PartialEq,
+    Eq,
+    Ord,
+    PartialOrd,
+    Hash,
+    Deref,
+    DerefMut,
+    Serialize,
+    Deserialize,
```

Comment:
Hmm, can you add attribute `#[serde(transparent)]`, so that `subspace_core_primitives::PublicKey` and this one would look the same in json?

Reply:
im ok with both, any advantages/disadvantages of what I did?